docs/source/DebuggingWithGDB.rst

```diff
@@ -185,3 +181,38 @@ breakpoints, executing commands and querying for output.
 
 When you check the output (``--show-job-log``) you can see that despite
 declaring the variable as 0, ff is injected and printed instead.
+
+
+rr wrapper
+----------
+
+It uses the ``--wrapper`` avocado functionality to run matching binaries
+inside the specified script. There is an example `rr` script in
+``/usr/share/avocado/wrappers/rr.sh`` which stores the information in
+test's `output_dir/rr` as `$bin_name.$serial_id`.
+
+Let's have a look at an example with malfunctioning qemu. Run the
+malfunctioning qemu inside avocado with wrapper::
+
+    $ avocado run "/usr/local/bin/qemu-system-aarch64 -machine virt" --wrapper /usr/share/examples/wrappers/rr.sh:*qemu*
```

Comment:
I mentioned this on meeting. I'd like to use `doublefree.py` as example, but without persistent tmpdir I can't. So I can use abstract test like `example.py` and mention in the documentation, that it should execute the binary we previously compiled with debugging symbols, or I can wait for the no-clenaup-tmpdir argument, or I can use this approach and explain it even now. Anyway I'd like to have an example in documentation along with the first mentioning of `rr` as it helps users to use the feature.